docs/proposals/files/trigger-deploy.yml

```diff
@@ -0,0 +1,30 @@
+
+name: TriggerTest
+
+on:
+  workflow_dispatch:
+    inputs:
+      cncf_project:
+        description: 'CNCF Project Name'
+        required: true
+        default: 'falco'
+      cncf_project_sub:
+        description: 'CNCF Project Subcomponent'
+        required: false
+        default: 'modern-ebpf'
+      version:
+        description: 'Version'
+        required: true
+        default: '0.37.0'
+
+jobs:
+  echo-inputs:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Echo Inputs
```

Comment:
Echo Inputs? is this for debugging? What are the inputs we echo - a project reference?

Reply:
The workflow was intended to be an example to help with reviewing the PR.

However I don't think its adding much and is a distraction so I've removed it